tst/standard/attr.tst

```diff
@@ -1174,6 +1174,19 @@ gap> for i in [1 .. 50] do
 > od;
 gap> DigraphOddGirth(gr);
 3
+gap> G := Digraph([[]]);
+<immutable digraph with 1 vertex, 0 edges>
+gap> for i in [2 .. 200] do
+>   G := DigraphAddVertex(G, i);
+>   G := DigraphAddEdges(G, [[1, i], [i, 1]]);
+> od;
+gap> D := CycleDigraph(7);
+<immutable digraph with 7 vertices, 7 edges>
+gap> for i in [1 .. 20] do
+>   D := DigraphDisjointUnion(D, G);
+> od;
+gap> DigraphOddGirth(D);
```

Comment:
Could you please do all of this test with mutable digraphs instead? It will remove a lot of copying I think and therefore quicker (and also using our cool new functionality 🙂): `G := Digraph(IsMutable[[]]);`, `D := DigraphMutableCopy(CycleDigraph(7));`, etc.